src/samples/article-poa/v2/complete.json

```diff
@@ -144,6 +144,12 @@
         ]
     },
     "dataSets": {
+        "availability": [
+            {
+                "type": "paragraph",
+                "text": "This is the text entered for the data availability text entry box."
```

Comment:
is this really the best example text? :thinking:

Reply:
Nope, but I don't have any examples of what this might look like.